scripts/gen_graphs.py

```diff
@@ -16,6 +16,7 @@ def generate_all_graphs_for_repos(all_repos):
         print(f"Generating graphs for repo {repo.name}")
         generate_solid_gauge_issue_graph(repo)
         generate_repo_sparklines(repo)
+        generate_donut_graph_line_complexity_graph(repo)
```

Comment:
Absolutely love that this is how easy it is to add new graphs to generate <3

Reply:
I'm trying to make it as maintainable as possible